docs/Requirements.md

```diff
@@ -5,71 +5,137 @@
 ### Debian-based requirements
 
 #### Ubuntu
-`sudo apt-get update && sudo apt-get install git cmake make gcc g++ clang libmysqlclient-dev libssl-dev libbz2-dev libreadline-dev libncurses-dev mysql-server libace-6.* libace-dev`
+
+```sh
+sudo apt-get update && sudo apt-get install git cmake make gcc g++ clang libmysqlclient-dev libssl-dev libbz2-dev libreadline-dev libncurses-dev mysql-server libace-6.* libace-dev
+```
+
+If you are using the `root` user
+
+```sh
+apt-get update && apt-get install git cmake make gcc g++ clang libmysqlclient-dev libssl-dev libbz2-dev libreadline-dev libncurses-dev mysql-server libace-6.* libace-dev
+```
```

Comment:
it's redundant, just say that if you are using root you don't need `sudo`

Reply:
Yes, it could be.